doc/contrib/pr.rst

```diff
@@ -3,6 +3,20 @@
 PR workflow
 ===========
 
+The PR, short for Pull Request, is a way to merge code on Github, where the main Arbor-repo is hosted.
+
+.. _contribpr-issue:
+
+Issues
+------
+
+New features, bugfixes or other kinds of contributions ideally start their lives as an Issue on our
```

Comment:
Issue -> issue

Reply:
Although a bit subtle, I capitalize when I'm referring to things in GH UI. You might have many issues, but until they are Issues, I probably don't know about them.

Reply:
I like the idea, but maybe highlight in a more obvious way then and explain it. Like written, it seems like a typo.

Reply:
See below.